src/ts/components/settingspanelitem.ts

```diff
@@ -32,7 +32,7 @@ export class SettingsPanelItem extends Container<ContainerConfig> {
 
     this.config = this.mergeConfig(config, {
       cssClass: 'ui-settings-panel-item',
-      role: 'menuitem',
+      role: this.config.role != null ? this.config.role : 'menuitem',
```

Comment:
The following should be sufficient? 🤔 This is just the default config and does not override what is in the `config`
```suggestion
      role: 'menuitem',
```

Reply:
The problem is that I need to override with config if there is a `menubar` present (bar with inputs inside of it). Since this component is used as a wrapper for input fields and as well for menu bars

Reply:
I don't get it. What is a `menubar` in this context?

Reply:
<img width="236" alt="Screenshot 2019-11-27 at 15 08 08" src="https://user-images.githubusercontent.com/15270879/69730098-c2557a80-1127-11ea-9957-744ae3f4a4cd.png">
<img width="289" alt="Screenshot 2019-11-27 at 15 08 16" src="https://user-images.githubusercontent.com/15270879/69730100-c2ee1100-1127-11ea-88f7-cde0b689779c.png">


These two are actually menu bars, they contain more than one control element

Reply:
Ah ok makes sense. But anyway once you set a config when initializing (e.g. in the `SubtitleSettingsPanelPage`) the role from this config will be applied. So no need to change the default value.